committees/project-director-election-process.md

```diff
@@ -0,0 +1,34 @@
+# Leadership Council _Project Director Election Process_ Committee
+
+## Goals
+
+* Design the election process for the next set of Rust Foundation project directors (ratified by the leadership council)
+* Clearly document and communicate the process and support its execution
+* Track, review, and document the effectiveness of the process
+
+## Delegated Responsibilities
+
+* None, this committee exists to create a proposal which will be consented to by the council
+
+## Duration
+
+* Temporary. This committee may be recreated each time we do a project director election in order to help support the process.
+
+## Contact, Communication, and Workspace
+
+- **Zulip Stream**: https://rust-lang.zulipchat.com/#narrow/stream/394329-council.2Fproject-director-election-proposal
+- **Meeting Minutes**: https://hackmd.io/@rust-leadership-council/BJANZIm_h
+
+## Process
+
+How does this committee work? Does it have meetings? Does it require consensus on all decisions? Does it present reports or recommendations to the Council? Is it required for decisions to be brought up to the Council?
+
+* This committee tracks it's backlog within the meeting minutes hackmd document.
+* This committee works via zulip and has video meetings on an as needed basis.
+* This committee operates via consent, the proposal will be presented to the council once all committee members have fully reviewed the proposal and have no further feedback or objections.
+
+## Members
+
+* Ryan Levick ([rylev](https://github.com/rust-lang/team/blob/master/people/rylev.toml))
+* Eric Holk ([eholk](https://github.com/rust-lang/team/blob/master/people/eholk.toml))
+* Jane Losare-Lusby ([yaahc](https://github.com/rust-lang/team/blob/master/people/yaahc.toml))
```

Comment:
Can you clarify that you mean having a GitHub issue tracking that? Can you say why that seems like it would be more fluid compared to a PR changing membership?

Reply:
Well, it depends how we track active committees and members of those. In the types team, we have these "initiative tracking issues" which represent ongoing work (https://github.com/rust-lang/types-team/issues?q=is%3Aissue+is%3Aopen+label%3Ainitiative-tracking-issue). If we had something similar for committees, we could assign members to the "tracking issue".

Reply:
while I do think this is good line of thought and I do want to see this go somewhere, I also wanna confirm my impression, this is more of an idle musing than a blocking objection to the proposal in the current state, right?

Reply:
Yes, ideal musing.